styles.css

```diff
@@ -0,0 +1,502 @@
+}
+
+.see-project-1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 20px;
+  background: #ff6b00;
+  margin-left: 20px;
+  margin-top: 8px;
+}
+
+.card-1 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 1581px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.card-2 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 1991px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.card-3 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 2441px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.card-4 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 2811px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.card-5 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 3221px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.card-6 {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 3631px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('./images/group2.png');
+}
+
+.container {
+  position: absolute;
+  width: 375px;
+  height: 1901px;
+  left: 0;
+  top: 4137px;
+  background: #1c1a19;
+}
+
+.about-heading {
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 79px;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-text {
+  width: 327px;
+  height: 148px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 24px;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: 24px;
+  margin-left: 24px;
+  margin-right: 190px;
+}
```

Comment:
[OPTIONAL]

- [ ] Please use the correct styling for the buttons, provided by the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) design.
- [ ] And also, try to implement the styling for the other states as well such as `active`, `hover` and `disabled`, the styling is already mentioned in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) design. 😉

![image](https://user-images.githubusercontent.com/20567503/176990640-5a257c59-fbb1-4726-bade-321b1ac53d3e.png)